### PR TITLE
update the base image to address cves

### PR DESCRIPTION
**Pull request by jaybeeunix**
This relates to https://issues.redhat.com/browse/OSD-7332

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/openshift/origin-cli:4.2.0
+FROM quay.io/openshift/origin-cli:4.8.0
```
Comment (lisa) [inline Dockerfile:1]: For the future, once we have some automation around it, would be to set the version to a build arg passed in at build-time.

**Review by lisa — APPROVED** [review]
/lgtm
/approve

File: Dockerfile (hunk 1 above)
Comment (katherinelc321) [reply Dockerfile:1]: I checked the repo and i saw an updated image tagged as "latest". Shouldn't we use that instead?

Comment (jaybeeunix) [reply Dockerfile:1]: yes, we should definitely use a more recent one. you can see what versions clair says are clear here: https://quay.io/repository/openshift/origin-cli?tab=tags

**Pull request merged**